worker/lease/manager.go

```diff
@@ -320,8 +320,8 @@ func (manager *Manager) retryingClaim(claim claim) {
 func (manager *Manager) handleClaim(claim claim) (bool, error) {
 	store := manager.config.Store
 	request := lease.Request{Holder: claim.holderName, Duration: claim.duration}
-	err := lease.ErrInvalid
-	action := "unknown"
+	var err error
```

Comment:
Variables seem to be given initial values... These seems to be lost with this change.